README.md

```diff
@@ -49,15 +49,35 @@ Matched sequence 👩🏿 — code points: 2
 Matched sequence 👩🏿 — code points: 2
 ```
 
-To match emoji in their textual representation as well (i.e. emoji that are not `Emoji_Presentation` symbols and that aren’t forced to render as emoji by a variation selector), `require` the other regex:
+## Regular expression flavors
+
+The package comes with three distinct regular expressions:
 
 ```js
-const emojiRegex = require('emoji-regex/text.js');
+// This is the recommended regular expression to use. It matches all
+// emoji recommended for general interchange, as defined via the
+// `RGI_Emoji` property in the Unicode Standard.
+// https://unicode.org/reports/tr51/#def_rgi_set
+// When in doubt, use this!
+const emojiRegexRGI = require('emoji-regex/RGI_Emoji.js');
+
+// This is the old regular expression, prior to `RGI_Emoji` being
+// standardized. In addition to all `RGI_Emoji` sequences, it matches
+// some emoji you probably don’t want to match (such as emoji component
+// symbols that are not meant to be used separately).
```

Comment:
When *should* you use this one? - comment reads as if answer is never

Reply:
It should only be used by those seeking 100% backwards-compatibility with older emoji-regex releases. I plan on removing it in the next major release.